alertmanager_gchat_integration/app.py

```diff
@@ -76,23 +101,16 @@ def post_alerts():
                 pass
         render_payload.update(alert)
         rendered_alert = J2_TEMPLATE_ENGINE.render(render_payload)
-
-        # Post request.
-        if CONFIG['app']['notification'].get('use_cards'):
-            post_request_data = {'cards': json.loads(rendered_alert)}
-        else:
-            post_request_data = {'text': rendered_alert}
-        post_request_details = requests.post(
-            notification_url,
-            json=post_request_data,
-            verify=False)
-
-        if post_request_details.ok:
-            LOGGER.info('Alert message posted successfully to GChat room <%s>.', room_name)
-        else:
-            LOGGER.error('Alert message failed to be posted to GChat room <%s>.', room_name)
-            LOGGER.error('%s - %s',
-                         str(post_request_details.status_code), str(post_request_details.text)
-                         )
+        if USE_CARDS:
+            rendered_alert = json.loads(rendered_alert, strict=False)
+        rendered_alert_list.append(rendered_alert)
+
+    if USE_CARDS:
+        post_request(notification_url, {'cards': rendered_alert_list})
+    elif group_alerts:
```

Comment:
Should `group_alerts` be applicable only when `USE_CARDS` is `true` ?
In that case, this elif will never be called.